### PR TITLE
Development branch to main branch PR

### PR DESCRIPTION
**Pull request by adamteddychang**
This is the final pull request for the Javascript CapStone, here are the features that we implemented:
- We set up the KanBan Board for this project
- We set up the project with Webpack and Jest
- We used an external API 
- We displayed the items from the API onto the document
- We made a popup for each individual item to display details and comments about the specific item
- We displayed the number of likes each item got from the API
- We made a function to add the likes and send them to the API
- We made a function to add a comment and send them to the API
- We displayed the comments that it got from the API
- We created counters for both the displayed items and comments, and we tested them. 
- We added final touches to the styling to make it aesthetically pleasing
- [Here is the link to our presentation](https://drive.google.com/file/d/1ytqk7WUwXJ_mL4xV-i_VagGo2Q-j37U8/view?usp=sharing)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- CSS
+- git
+- Javascript
+- API
+
+
+## Live Demo
+
+
+[site](https://taher-web-dev.github.io/JS_Cap/)
```
Comment (Dipeshtwis) [inline Readme.md:24]: - [ ] Looks like the live demo link is not working perfectly. I am seeing this only (shown in the below screenshot). Please update the live demo link with the correct link or setup.


Screenshot from 2021-11-18 23-53-59

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @adamteddychang and @Taher-web-dev, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Your project is working fine.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] I would recommend removing the unnecessary file from the Github, Please add it to the `.gitignore` file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adamteddychang** [comment]
> 
> * [ ]  I would recommend removing the unnecessary file from the Github, Please add it to the `.gitignore` file.
> 
Hello, thank you for your review. 
I'm sorry but I am unsure what you meant by unnecessary files. 
Can you please elaborate? Thank you

**Comment by Dipeshtwis** [comment]
> > * [ ]   I would recommend removing the unnecessary file from the Github, Please add it to the `.gitignore` file.
> 
> Hello, thank you for your review. I'm sorry but I am unsure what you meant by unnecessary files. Can you please elaborate? Thank you

Sorry, I missed mentioning that file. It is `.gitattributes`.

**Comment by adamteddychang** [comment]
> > > * [ ]    I would recommend removing the unnecessary file from the Github, Please add it to the `.gitignore` file.
> > 
> > 
> > Hello, thank you for your review. I'm sorry but I am unsure what you meant by unnecessary files. Can you please elaborate? Thank you
> 
> Hello @adamteddychang and @Taher-web-dev,
> 
> Sorry, I missed mentioning that file. It is `.gitattributes`. Thanks for reminding me.

Thank you for letting me know!

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Good job addressing the requested changes from the previous CR 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**